### PR TITLE
Adds bookshelf project

### PR DESCRIPTION
**Pull request by alismith-dev**
Adds a [bookshelf project](https://github.com/CodeYourFuture/bookshelf-project). Adapted from an old project repo to be consistent with the group-projects style. Feel free to open an issue/PR against that repo if you think it needs some changes.

I've made a (somewhat messy!) example implementation [here](https://github.com/CodeYourFuture/bookshelf-project/pull/39).

### CODE REVIEW

**Comment by alismith-dev** [comment]
@NateWr could you take a look at this? Finally got around to merging the changes into the bookshelf project repo.

File: bookshelf/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Bookshelf App
+
+In this project, you will build a bookshelf app. The app structure and design have been put in place. To complete the app, you'll need to add functionality so that read and unread books can be shown and the count updated.
```
Comment (SallyMcGrath) [inline bookshelf/readme.md]:
```suggestion
In this project, you will build a bookshelf app. The app structure and design are done. To complete the app, add functionality to: 
- show both read and unread books
- update the count of read books.
```

active voice, simpler sentences

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Tiny changes to wording

**Comment by alismith-dev** [comment]
Thanks. Waiting until https://github.com/CodeYourFuture/group-project-bookshelf/pull/43 is approved before merging this.

**Pull request merged**